src/privileged/multipreffer/api.js

```diff
@@ -117,7 +78,7 @@ this.FirefoxHooks = {
     if (addon.id !== gExtension.id) {
       return;
     }
-    await this.cleanup();
```

Comment:
no need to have the function be async anymore (GitHub doesn't allow me to comment on the relevant line, ugh)